### PR TITLE
Add Weekly Change

### PR DESCRIPTION
**Pull request by LucasFares**
Adds percent changes to compare graph to show user if energy is being saved.

### CODE REVIEW

File: src/client/app/containers/CompareChartContainer.js [hunk 1]
```diff
@@ -26,6 +26,22 @@ function mapStateToProps(state) {
 	// Power used up to this point last week
 	let currentLastWeek = 0;
 	const soFar = moment().diff(moment().startOf('week'), 'days');
+	const delta = function (change) {
```
Comment (johndiiorio) [inline src/client/app/containers/CompareChartContainer.js]: You should write this as an arrow function

File: src/client/app/containers/CompareChartContainer.js [hunk 2]
```diff
@@ -26,6 +26,22 @@ function mapStateToProps(state) {
...
+		if (isNaN(change)) {
+			return '';
+		}
+		if (change < 0) {
+			return `You have used ${parseInt(change.toFixed(2).replace('.', ''), 10)}% less energy this week.`;
+		}
+		return `You have used ${parseInt(change.toFixed(2).replace('.', ''), 10)}% more energy this week.`;
+	};
+
+	const colorize = function (change) {
```
Comment (johndiiorio) [inline src/client/app/containers/CompareChartContainer.js]: Arrow function

File: src/client/app/containers/CompareChartContainer.js [hunk 3]
```diff
@@ -26,6 +26,22 @@ function mapStateToProps(state) {
...
+		if (change < 0) {
```
Comment (johndiiorio) [inline src/client/app/containers/CompareChartContainer.js]: Shouldn't this be <= 0? Or is no change not great?

File: src/client/app/containers/CompareChartContainer.js [hunk 4]
```diff
@@ -26,6 +26,22 @@ function mapStateToProps(state) {
 	// Power used up to this point last week
 	let currentLastWeek = 0;
 	const soFar = moment().diff(moment().startOf('week'), 'days');
+	const delta = function (change) {
+		if (isNaN(change)) {
+			return '';
+		}
+		if (change < 0) {
+			return `You have used ${parseInt(change.toFixed(2).replace('.', ''), 10)}% less energy this week.`;
```
Comment (johndiiorio) [inline src/client/app/containers/CompareChartContainer.js]: You should replace "You have used" with "{Meter name} has used {value}% less energy this week"

File: src/client/app/containers/CompareChartContainer.js [hunk 5]
```diff
@@ -26,6 +26,22 @@ function mapStateToProps(state) {
...
+		}
+		return `You have used ${parseInt(change.toFixed(2).replace('.', ''), 10)}% more energy this week.`;
```
Comment (johndiiorio) [inline src/client/app/containers/CompareChartContainer.js]: You should replace "You have used" with "{Meter name} has used {value}% more energy this week"

File: src/client/app/containers/CompareChartContainer.js (hunk 3 above)
Comment (LucasFares) [reply src/client/app/containers/CompareChartContainer.js]: We want them to improve!

**Pull request merged**